src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -1345,7 +1385,7 @@ public void testDiagramsAndGraphics() {
 
         //get the loads map data of a network
         webTestClient.get()
-            .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-map/loads/", studyNameUserIdUuid, rootNodeUuid)
+            .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-map/loads", studyNameUserIdUuid, rootNodeUuid)
```

Comment:
Why remove ending '/' in the URL? it's not removed elsewhere

Reply:
it triggered me but not enough to delete them all... so reverted :)